release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -1,41 +1,141 @@
-- If a dashboard isn't listed here, or a test isn't on one of the listed dashboards, **_I'm not looking at it_**
+## Overview of CI Signal responsibilities
+
+CI Signal lead assumes the responsibility of the quality gate for the release. This person is responsible for:
+- Monitoring various e2e tests in sig-release dashboards ([master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking), [master-upgrade](https://k8s-testgrid.appspot.com/sig-release-master-upgrade), [x.y-blocking](https://k8s-testgrid.appspot.com/sig-release-1.11-blocking)) on a regular basis and providing early and continuous signals on release and test health to both Release team and SIGs
+- Ensuring that all release blocking tests provide a clear Go/No-Go signal for the release
+- Flagding regression as close to source as possible i.e., as soon as the offending code was merged
+- Filing issues proactively for test failures and flakes, triaging issues to appropriate SIG/owner, following up on status and verifying fixes on newer test runs
+- Studying patterns/history of test health from previous releases and working closely with SIGs and test owners to 
+  - Understand and fix the test for the current release
+  - Understand if the test needs to be release blocking
+  - Advise Release team in making informed go/no-go calls 
+  - Work with SIG-Testing on any possible test infra improvement to help improve test pass rate
+- Making recommendations to SIG-Release for promoting and demoting release blocking and merge blocking tests as per the [Blocking tests Proposal](https://docs.google.com/document/d/1kCDdmlpTnHPQt5z8JzODdFCc3T2D4MKR53twsDZu20c/edit)
+
+### Explicit detail is important:
+
+- If you're looking for answer that's not in this document, please file an issue so we can keep the document current.
+- Generally CI Signal lead errs on the side of filing an issue for each test failure or flake before following up with SIGs/owners. This way we dont _lose track of an issue_
+- If a dashboard isn't listed here, or a test isn't on one of the listed dashboards, **_CI Signal lead is not looking at it_**
 
 ## Requirements
-- I need the ability to notify kubernetes org github teams, so I must be a kubernetes org collaborator or member
+
+- Signal lead needs the ability to notify kubernetes org github teams, so must be a kubernetes org collaborator or member
   - The process to become one of these is in [our community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#requirements-for-outside-collaborators)
-- I need the ability to add a milestone to issues, so I must be a member of the [kubernetes-pm github team](https://github.com/orgs/kubernetes/teams/kubernetes-pm/members)
+- Signal lead needs the ability to add a milestone to issues, so must be a member of the [kubernetes-pm github team](https://github.com/orgs/kubernetes/teams/kubernetes-pm/members)
```

Comment:
Should now be https://github.com/orgs/kubernetes/teams/kubernetes-milestone-maintainers